Dockerfile

```diff
@@ -18,4 +42,8 @@ RUN mv grafterm-linux-amd64 /usr/bin/grafterm
 
 RUN chmod 755 /usr/bin/grafterm
 
+RUN chmod u+s /bin/ping
+
+USER 65534:65534
+
 CMD [ "ping", "cray-sysmgmt-health-promet-prometheus" ]
```

Comment:
Just to my knowledge... Could you please explain how this image is used? Trying to understand why do we use ping as cmd entry.

Reply:
With this grafterm docker image, we are not actually creating any service. We're only using it to run a binary (grafterm) that will work with prometheus to plot the graphs.
When we start up the pod, I just used the ping here to retrieve some logs to make sure the Prometheus is reachable.